src/components/card/ProfileCard.jsx

```
@@ -1,7 +1,12 @@
 import React from "react";
 import { useSelector } from "react-redux";
 
-const ProfileCard = ({ title, subTitle, image , subTitleSize= "text-base" }) => {
+const ProfileCard = ({
+  title,
```

ESLint

[error] 'title' is missing in props validation

(react/prop-types)